Model/Import/Attributes.php

```diff
@@ -86,19 +232,28 @@ public function __construct(
         $this->cacheType = $cacheType;
         $this->massProdValues = $massProdValues;
 
-        $this->mappingTable = $this->getConnection()->getTableName('sinch_restrictedvalue_mapping');
-        $this->cpeTable = $this->getConnection()->getTableName('catalog_product_entity');
-        $this->filterCategoriesTable = $this->getConnection()->getTableName('sinch_filter_categories');
+        $this->mappingTable = $this->resourceConn->getTableName('sinch_restrictedvalue_mapping');
+        $this->cpeTable = $this->resourceConn->getTableName('catalog_product_entity');
+        $this->filterCategoriesTable = $this->resourceConn->getTableName('sinch_filter_categories');
 
         $writer = new \Zend\Log\Writer\Stream(BP . '/var/log/sinch_attributes.log');
         $logger = new \Zend\Log\Logger();
         $logger->addWriter($writer);
         $this->logger = $logger;
+        $this->output = $output;
     }
 
+    /**
+     * @param $categoryFeaturesFile
+     * @param $restrictedValuesFile
+     * @param $productFeaturesFile
+     * @throws \Magento\Framework\Exception\InputException
+     * @throws \Magento\Framework\Exception\StateException
+     */
     public function parse($categoryFeaturesFile, $restrictedValuesFile, $productFeaturesFile)
     {
-        $this->logger->info("--- Begin Attribute Parse ---");
+        $this->output->writeln("---- Begin Attribute Parse ----");
```

Comment:
These pairs of function calls could be condensed into a function, seeing as they output the same thing every time.